SocketRocket.xcodeproj/project.pbxproj

```diff
@@ -705,6 +703,7 @@
 				);
 				OTHER_LDFLAGS = "-ObjC";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PUBLIC_HEADERS_FOLDER_PATH = include/$PRODUCT_NAME;
```

Comment:
nit: The "standard" way to write this (as defined by those created by Xcode) is using parentheses: `$(PRODUCT_NAME)`. Not a blocker though.